lib/errors.js

```diff
@@ -168,12 +168,14 @@ class UnprocessableEntity extends HttpClientError {
 exports.UnprocessableEntity = UnprocessableEntity;
 
 class DeploymentAlreadyLocked extends UnprocessableEntity {
-  constructor(deploymentName, lockInfo) {
-    let lockBy = '';
+  constructor(deploymentName, lockInfo, lockMessage) {
+    let message = `Deployment ${deploymentName} ${CONST.OPERATION_TYPE.LOCK}`;
     if (lockInfo) {
```

Comment:
not using lockMessage in case of lockInfo is present? Is lockMessage undefined in this case ?

Reply:
Yes. The order of precendence is:
1. LockInfo
2. LockMessage
If LockInfo is present then message is built from it. If its not present then whatever lock message is provided is used. If both are undefined then default message is set. (Line 172)